.azure-pipelines/jobs/prod-build.yml

```diff
@@ -1,6 +1,6 @@
 steps:
   - template: ../steps/install-nodejs.yml
   - template: ../steps/install-dependencies.yml
-  - script: yarn build
+  - script: yarn build && cd dist && list -l && cd ..
```

Comment:
Forgot to remove this?

Reply:
no, I want show the result on CI, then remove it